docs/admin/servers/lib/LIB-MS.md

```diff
@@ -46,19 +65,36 @@ Replace the default values the appropriate values for your setup.
 ### Start Microservice
 
 ```bash
-yarn install
-yarn build
-yarn start
+yarn install    # Install dependencies for the microservice
+yarn build      # build the application
+yarn start      # start the application
+```
+
+You can press `Ctl+C` to halt the application. If you wish to run the microservice in the background, use
+
+```bash
+nohup yarn start & disown
 ```
 
 The lib microservice is now running and ready to serve files, functions, and models.
 
 Users can access the library microservice at URL: `http://localhost:<PORT>/lib`.
 
+## Developer Commands
```

Comment:
Please remove the **Developer Commands** section completely. Keep the next section - **Service Endpoint**.